Dockerfile

```diff
@@ -1,4 +1,6 @@
-FROM gcr.io/distroless/static:debug
+FROM gcr.io/distroless/static:debug as builder
+FROM gcr.io/distroless/static:nonroot
```

Comment:
Perhaps we use `debug-nonroot` tag here?

Reply:
Today I learned this was an image, thanks @joelanford  😄

Reply:
Yeah those distroless images are super difficult to discover. It's a lot of trial and error for me.